src/bp/admin/ui/src/app/Header/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Alignment, Button, Colors, Icon, Navbar, Tooltip } from '@blueprintjs/core'
+import { lang, ShortcutLabel } from 'botpress/shared'
+import cx from 'classnames'
+import React, { FC } from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import WorkspaceSelect from '~/app/WorkspaceSelect'
+
+import AccessControl from '~/auth/AccessControl'
+import UserDropdownMenu from '~/user/UserDropdownMenu'
+import { AppState } from '../rootReducer'
+import { toggleBottomPanel } from '../uiReducer'
+import style from './style.scss'
+
+type Props = ConnectedProps<typeof connector>
+
+const Header: FC<Props> = props => {
+  return (
+    <header className={cx('bp-header', style.header)}>
+      <Navbar>
+        <Navbar.Group>
+          <Navbar.Heading>
+            <div className={cx('bp-sa-title', style.title)}>
+              <span>{props.pageTitle || ''}</span>
+              {props.pageHelpText && (
+                <Tooltip content={props.pageHelpText}>
```

Comment:
Yeah I saw the deprecation notice, but it still works correctly and we use `Tooltip` in numerous places. This would also require us to add a reference to @blueprintjs/popover2 to the studio, admin, module builder, and pretty much everywhere else.

We will keep Tooltip until we need to upgrade to a version of blueprintjs which removes it, then we'll be able to upgrade every usage of Tooltip at the same time